### PR TITLE
Custom update interval and new variables for subscription

### PR DESCRIPTION
**Pull request by kastov**
Add following headers:
- Support-Url
- [Profile-Web-Page-Url ](https://docs.cfw.lbyczf.com/contents/urlscheme.html#profile-web-page-url)
- Profile-Title

New .env variables SUB_PROFILE_TITLE, SUB_SUPPORT_URL, SUB_UPDATE_INTERVAL

So you can set your own Profile-Title, Support-Url and Update Interval (default is 12 hours)

### CODE REVIEW

File: app/utils/share.py
Comment (M03ED) [inline app/utils/share.py:1]: i think you need to update this file , in last commits this file changed a lot

Comment (kastov) [reply app/utils/share.py:1]: Okay. No problem. I will close this PR and create a new one, based on latest commits

Comment (kastov) [reply app/utils/share.py:1]: Not sure that i merged correct. But seems like now conflict resolved.

**Comment by septs** [comment]
is there any documentation on the `profile-title` and `support-url` header fields?

**Pull request merged**